bin/check-database-roles.sh

```diff
@@ -16,6 +16,8 @@ environment=$2
 
 ./bin/terraform-init.sh "infra/$app_name/database" "$environment"
 db_role_manager_function_name=$(terraform -chdir="infra/$app_name/database" output -raw role_manager_function_name)
+db_config=$(terraform -chdir="infra/$app_name/database" output -json db_config)
+payload='{"action":"check","config":'$db_config'}'
```

Comment:
The quoting here looks a little strange. It looks like you aren't quoting the `$db_config` var. I think you want something like this, which is less legible but is more shell safe.
```suggestion
payload="{\"action\":\"check\",\"config\":\"${db_config}\"}" 
```

Reply:
Ah, thank you!